internal/super/generate.go

```diff
@@ -102,7 +102,18 @@ func generateContract(
 	_ flowkit.Services,
 	state *flowkit.State,
 ) (result command.Result, err error) {
-	return generateNew(args, "contract", generateFlags.Directory, logger, state)
+	if len(args) < 1 {
```

Comment:
shouldn't line 86 cover that already?

https://github.com/onflow/flow-cli/pull/1566/files#diff-1a34fb00f194ec131896919bfad4b3fdaf639c76fd91304228c346eb73dc89b9L86

Reply:
Yes good catch! Removed